examples/siro_sandbox/sandbox_app.py

```diff
@@ -147,19 +147,33 @@ def __init__(self, args, config, gui_input):
         # self._target_obj_ids = None
         # self._goal_positions = None
 
-        self.env_reset()
-        self._prev_post_sim_update_dict = {}
+        self._env_reset()
 
-    def env_reset(self):
+    def _env_reset(self):
         self._obs = self.env.reset()
         self._metrics = self.env.get_metrics()
-        self.on_environment_reset()
+        self._on_environment_reset()
 
-    def env_step(self, action):
+    def _env_step(self, action):
         self._obs = self.env.step(action)
         self._metrics = self.env.get_metrics()
 
-    def on_environment_reset(self):
+    def _env_episode_active(self) -> bool:
+        """
+        Returns True if current episode is active:
+        1) not self.env.episode_over - none of the constraints is violated, or
+        2) not self._env_task_complete - success measure value is not True
+        """
+        return not (self.env.episode_over or self._env_task_complete)
+
+    def _compute_action_and_step_env(self):
```

Comment:
Nit: when a function like this maybe or maybe won't do something, I prefer `try_` or `check_` in the name. I.e. `_check_compute_action_and_step_env`.